devices/milight.coffee

```diff
@@ -28,18 +28,32 @@ module.exports = (env) ->
 
     turnOn: ->
       @_updateState power: true
-      @device.sendCommands(nodeMilight.commands.rgbw.on(@zone))
-      if @mode
-        color = Color(@color).rgb()
-        @device.sendCommands(nodeMilight.commands.rgbw.rgb255(color.r, color.g, color.b))
+
+      if @config.type == "White"
+
+        @device.sendCommands(nodeMilight.commands.white.on(@zone))
+        @device.sendCommands(nodeMilight.commands.white.maxBright())
```

Comment:
Why `maxBright` and not `brightness(@brightness)` like we do it in case of rgb bulb?